src/locales/en.json

```diff
@@ -389,6 +389,7 @@
     "empty": "No connectors configured",
     "noAccount": "No account",
     "add": "Add a service",
+    "update": "Update",
```

Comment:
Update available would be better here. "Update" would be for a button.

Reply:
I prefer to keep a short text here since we don't have so much space, we prefer to keep the most space to display correctly with the account name